python/dronin/telemetry.py

```diff
@@ -193,23 +197,70 @@ def __handle_handshake(self, obj):
 
             if obj.Status == obj.ENUM_Status['Disconnected']:
                 # Request handshake
-                print("Disconnected")
+                #print("Disconnected")
```

Comment:
These should probably all use python logging so they can be enabled at runtime... For another PR. :)